src/database/dbHelper.js

```diff
@@ -12,67 +6,220 @@ const typeorm = require('typeorm');
 const dbName = '/srce.db';
 const pathToDatabase = path.join(__dirname, dbName);
 
-// Entitys
-const Person = require('./entity/Person').Person;
-
-function checkIfDatabaseExists() {
-    fs.access(pathToDatabase, fs.F_OK, e => {
-        if (e) {
-            module.exports.CreateDatabase();
-        } else {
-            console.log('Already created');
-        }
-    });
-}
+const Sequelize = require('sequelize');
+const sequelize = new Sequelize({
+    dialect: 'sqlite',
+    storage: pathToDatabase
+  });
+
+
+  //testing the conection
+sequelize
+.authenticate()
+.then(() => {
+  console.log('Connection has been established successfully.');
+})
+.catch(err => {
+  console.error('Unable to connect to the database:', err);
```

Comment:
Don't just throw an error, try to recreate DB. Check if file exists, if not create a new one and do "migrations" before starting an application.